packaging/nuget/NServiceBus.Ninject.nuspec

```diff
@@ -13,9 +13,9 @@
     <description>The Ninject Container for the nservicebus</description>
     <releaseNotes></releaseNotes>
     <copyright>$copyright$</copyright>
-    <tags>nservicebus servicebus msmq cqrs publish subscribe</tags>
+    <tags>$tags$</tags>
     <dependencies>
-      <dependency id="NServiceBus" version="[5.0.0, 6.0.0)" />
+      <dependency id="NServiceBus" version="6.0.0-beta0001" />
```

Comment:
We had to do that in the past due to "unstable", since we've move to alpha we should be able to do ranges like @SimonCropp proposes